<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#buscar_button, QPushButton#logout_button {
	background-color: rgba(0, 0, 0, 0);
	color:rgba(85, 98, 112, 255);
    border-radius: 20px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#buscar_button:hover,  QPushButton#logout_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 240),
        stop:1 rgba(220, 220, 220, 80)
    );
	color:rgba(155, 168, 182, 220);
}

QPushButton#buscar_button:pressed, QPushButton#logout_button:pressed {
	padding-left:5px;
	padding-top:5px;
	color:rgba(115, 128, 142, 255);
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 230),
        stop:1 rgba(180, 180, 180, 50)
    );
}

QPushButton#cliente_button, QPushButton#compras_button, QPushButton#sobre_button, QPushButton#produtos_button {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(15, 35, 70, 120), 
        stop:1 rgba(200, 200, 200, 40)
    );
    color: rgba(255, 255, 255, 180);  
    border-radius: 5px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#cliente_button:hover, QPushButton#compras_button:hover, QPushButton#sobre_button:hover,
QPushButton#produtos_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 150),
        stop:1 rgba(220, 220, 220, 50)
    );
}

QPushButton#cliente_button:pressed, QPushButton#compras_button:pressed, QPushButton#sobre_button:pressed,
QPushButton#produtos_button:pressed {
    padding-left: 3px;
    padding-top: 3px;
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 150),
        stop:1 rgba(180, 180, 180, 30)
    );
}
</string>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>711</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/log_background.png);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="barra_pesquisa">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>448</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 20px;
    color: rgba(255, 255, 255, 230);
    padding: 7px 13px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}

</string>
    </property>
    <property name="placeholderText">
     <string>Informe caracter√≠stica do produto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buscar_button">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>448</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Pesquisar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/ic_busca.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Pesquisar produto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fundo_transparente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba( 0 ,0, 0, 100 );
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sobre_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="produtos_button">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Produto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compras_button">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrinho</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cliente_button">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ciente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Logout</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/ic_logout.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba( 0 ,0, 0, 100 );
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>50</y>
      <width>450</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>fundo_transparente</zorder>
   <zorder>label</zorder>
   <zorder>buscar_button</zorder>
   <zorder>barra_pesquisa</zorder>
   <zorder>label_3</zorder>
   <zorder>logout_button</zorder>
   <zorder>label_2</zorder>
   <zorder>sobre_button</zorder>
   <zorder>compras_button</zorder>
   <zorder>cliente_button</zorder>
   <zorder>produtos_button</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
